### ISSUE TITLE
Bug: cant build Java examples

### ISSUE THREAD

**Issue by PedroEsnaola** [microsoft/semantic-kernel#7091]
**Describe the bug**
O think that there is an inconsistency between the examples and the actual filenames

Even when installing all available maven deps, I wasn't able to find the SKBuilder class to actually build the Kernels, therefore leaving me unable to test anything

Did any changes happened recently that leaves the examples obsolete?

iReproduce**

**Expected behavior**
The samples in the project should work

**Screenshots**

**Platform**
 - OS: [e.g. Windows, Mac]
 - IDE: [e.g. Visual Studio, VS Code]
 - Language: [e.g. C#, Python]
 - Source: [e.g. NuGet package version 0.1.0, pip package version 0.1.0, main branch of repository]

**Additional context**

**Comment by johnoliver** [comment]
Yes there has been a fair amount of movement in this area, the samples at the new location: https://github.com/microsoft/semantic-kernel-java/tree/main/samples should build fine. Please let us know if you have any issues with them
